pkgs/top-level/all-packages.nix

```
@@ -33282,6 +33282,8 @@ with pkgs;
 
   ### MISC
 
+  argononed = callPackage ../misc/drivers/argononed { };
```

```suggestion
  argononed = callPackage ../misc/drivers/argononed { };
```
This is out of alphabetical order.

---

@Misterio77 Can you fix this? Otherwise LGTM.